app/Models/User.php

```diff
@@ -67,4 +67,14 @@ public function orders()
     {
         return $this->hasMany('App\Models\Order', 'user_id', 'id');
     }
+
+    /**
+     * Check if user is Admin
+     *
+     * @return boolean
+     */
+    public function isAdmin()
+    {
+        return $this->role;
```

Comment:
return $this->role == User::ADMIN_ROLE hay gì đó để sau này có mở rộng thêm role thì dễ quản lý em

Reply:
dạ okay ạ anh.